tests/em/static/test_DC_Utils.py

```diff
@@ -0,0 +1,341 @@
+from __future__ import print_function
+import unittest
+import numpy as np
+import matplotlib.pyplot as plt
+from SimPEG.EM.Static import DC, Utils as DCUtils
+from SimPEG import Mesh, Maps
+from SimPEG.Utils import io_utils
+import shutil
+import os
+
+try:
+    from pymatsolver import Pardiso as Solver
+except ImportError:
+    from SimPEG import SolverLU as Solver
+
+
+class DCUtilsTests_halfspace(unittest.TestCase):
+
+    def setUp(self):
+        url = 'https://github.com/thast/Benchmarks_files/raw/master/Test_DCUtils/'
```

Comment:
Should these be moved to simpeg benchmark repo?

Reply:
@rowanc1: that's the plan. It was just to be able to run the tests while waiting the file to be uploaded on the simpeg benchmark repo. There are now so this will be change shortly.